konfera/models/ticket.py

```diff
@@ -58,4 +58,12 @@ def save(self, *args, **kwargs):
             order = Order(price=self.type.price, discount=discount, status=AWAITING, purchase_date=timezone.now())
             order.save()
             self.order = order
-        super(Ticket, self).save(*args, **kwargs)
+            super(Ticket, self).save(*args, **kwargs)
+        else:
+            super(Ticket, self).save(*args, **kwargs)
```

Comment:
I wanted to do that, but decided against it.
In case an order has to be created first, there will only be one ticket, so that the price and discount are the values of the one ticket. 
If however an order exists, we don't actually know if the current ticket is new or is being edited. We would have to loop over the tickets and exclude the current ticket if it exists. 
By saving the ticket first, we can be sure that the we calculate the correct price. If you want I can add a comment about that.